### PR TITLE
FR: All coasters translated

### PR DESCRIPTION
**Pull request by rvgr**
Translated all coasters name and description as in RCT2.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I cannot merge this without rebase. If you can't rebase, you can also open a new PR. (Make sure you don't overwrite the strings you changed with the other PR.)
Also, how RCT2 translated strings is not important. Many RCT2 translations are horrendous anyway.

File: data/language/french.txt [hunk 1]
```diff
@@ -5,32 +5,32 @@ STR_0000    :
 STR_0001    :{STRINGID} {COMMA16}
 STR_0002    :Montagnes russes en spirales
 STR_0003    :Montagnes russes debout
-STR_0004    :Montagnes russes balancantes suspendues 
+STR_0004    :Montagnes russes suspendues 
 STR_0005    :Montagnes russes inversées
-STR_0006    :Montagnes russes junior
+STR_0006    :Mini-montagnes junior
```
Comment (Gymnasiast) [inline data/language/french.txt]: Seems a strange translation to me: "Mini junior roller coasters". There are junior roller coasters and mini coasters, but no mini junior roller coaster.

Comment (rvgr) [inline data/language/french.txt]: That's the original translation in RCT2. Junior roller coasters are translated as "Mini-montagnes junior" and mini coasters as "Mini-montagnes russes".

Comment (Gymnasiast) [inline data/language/french.txt]: How RCT2 translated it is not important. The goal is having a good translation, not a translation that is the same as RCT2.

**Comment by rvgr** [comment]
Well, I think using original RCT2 translations will help us complete a 100% translation (at least of non-OpenRCT2 strings) quite fast, and is an objective base on which we could improve upon later. Translations are prone to subjective interpretations that need time to be discussed, so having this base already in place would be nice.

Anyway, quite new to github. What do you mean by rebase?

**Comment by Gymnasiast** [comment]
Your predecessor already translated Junior coaster, and that translation is clearer than the original RCT2 one. That's why I want to keep it.

You based this PR on a previous version of the french.txt file. Since the file has been changed in the meantime (by your other PR) you have to do a `git rebase` command in order to merge it. You cannot do this on Github, you need to have Git running on your own computer.

Considering you're new to this, it's way easier to just create a new PR.

**Pull request closed without merging**